code/game/objects/items/stacks/packagewrap.dm

```diff
@@ -21,8 +21,8 @@
 		/obj/item/weapon/gift,
 		/obj/item/weapon/winter_gift,
 		/obj/item/weapon/storage/evidencebag,
-		/obj/item/weapon/legcuffs/bolas,
-		/obj/item/weapon/storage
```

Comment:
not going to be able to store wrap in backpacks anymore unless you click the UI

Reply:
You can click an empty slot in the backpack, or move while you're wrapping to put the wrap in the backpack. That's why the ` do_after()` is there.

Reply:
>move it there while you're wrapping

won't that cancel the wrap because your held item has changed?
also it seems unintuitive to work this way, imagine trying to explain to someone IC that the only way to insert wrapping paper into a backpack is to view the UI

Reply:
No, it'll cancel the wrap because you break out of the do_after.
It's not intuitive, but it's the same shit as trying to recycle a wrench in an autolathe.
It just works.